app/assets/config/uswds_rails_manifest.js

```diff
@@ -1,2 +0,0 @@
-//= link_directory ../javascripts/uswds/rails .js
```

Comment:
agreed, it does seem odd. It was generated by Rails as boilerplate. If tests pass w/o it, I'm happy to see it deleted.